### ISSUE TITLE
emmet not woking well in vscode

### ISSUE THREAD

**Issue by littlee** [csstools/postcss-language#9]
## in postcss mode, typing 'col' always has only ONE suggestion
![image](https://user-images.githubusercontent.com/7534668/123541460-501ba080-d777-11eb-899e-70de0202d35b.png)

## in css mode, typing 'col' gets MANY suggestions, which is good.
![image](https://user-images.githubusercontent.com/7534668/123541447-42661b00-d777-11eb-8326-69f28743b9ab.png)

**Comment by LeaVerou** [comment]
For me Emmet is not working at all in PostCSS. E.g. I type `fz`(Tab) and nothing happens, whereas it works fine in CSS.

**Comment by honi** [comment]
I'm having the same issue as well. Has anybody found a solution?

**Comment by ZackPlauche** [comment]
Yeah, emmet literally doesn't use any of the builtin css stuff once this plugin is installed. I just uninstalled it and now my css files work again, and don't give me @apply or whatever errors. 🤷‍♂️. Maybe just the tailwind plugin is good enough.

**Comment by chriskoelle** [comment]
you can fix this by adding `postcss` to the `emmet.includeLanguages` setting in your vscode preferences:
```json
{
  "emmet.includeLanguages": {
    "postcss": "css"
  }
}
```

https://marketplace.visualstudio.com/items?itemName=csstools.postcss#adding-support-for-emmet

**Comment by kushagra-unorg** [comment]
> you can fix this by adding `postcss` to the `emmet.includeLanguages` setting in your vscode preferences:
> 
> ```json
> {
>   "emmet.includeLanguages": {
>     "postcss": "css"
>   }
> }
> ```
> 
> https://marketplace.visualstudio.com/items?itemName=csstools.postcss#adding-support-for-emmet

After doing this, the emmet is working but it is giving only some suggestions

**Comment by mnzulfahmi17** [comment]
> > you can fix this by adding `postcss` to the `emmet.includeLanguages` setting in your vscode preferences:
> > ```json
> > {
> >   "emmet.includeLanguages": {
> >     "postcss": "css"
> >   }
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > https://marketplace.visualstudio.com/items?itemName=csstools.postcss#adding-support-for-emmet
> 
> After doing this, the emmet is working but it is giving only some suggestions

Same, not working fully yet, but this helpfull for now.. i hope more solution to fixed and working fully

**Comment by jorgeart81** [comment]
Adding `css `to the `files.associations` setting in your settings.json:

}
`  "files.associations": {
    "*.css": "css"
  },`
